components/properties/src/props.rs

```diff
@@ -89,6 +90,7 @@ impl BidiClass {
 /// It does not support grouped categories (eg `Letter`). For grouped categories, use [`GeneralCategoryGroup`].
 #[derive(Copy, Clone, PartialEq, Eq, Debug, Ord, PartialOrd)]
 #[cfg_attr(feature = "serde", derive(Serialize, Deserialize))]
+#[allow(clippy::exhaustive_enums)] // this type is stable
```

Comment:
unsure about this. I think GC is a closed set?

Reply:
What does the spec say?

I think we made `GeneralCategory` an `enum` rather than a newtype because it is stable, implying that the enum should be exhaustive.

Reply:
https://www.unicode.org/policies/stability_policy.html#Property_Value

stable